src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/AppSettings.cs

```diff
@@ -36,7 +36,10 @@ public class EmailSettings
     {
         public string Host { get; set; }
         public int Port { get; set; }
+        public string Username { get; set; }
+        public string Password { get; set; }
         public string DefaulFromEmail { get; set; }
         public string DefaultFromName { get; set; }
+        public bool HasCredential => !string.IsNullOrEmpty(Username) && !string.IsNullOrEmpty(Password);
```

Comment:
don't use the `not` operator (`!`) at the start of a boolean expression, it's not clear. instead, use a comparison with `false`.
for example, here it can be changed to this:
```cs
public bool HasCredential => (string.IsNullOrEmpty(Username) is false) && (string.IsNullOrEmpty(Password) is false);
```